### PR TITLE
Select page after detaching background media

### LINKED ISSUE #6907: Keep Page selected after removing Background Media
## Feature Description
Currently, when background media is removed from the Page, the media gets selected, however, the Page should be selected after that.

See [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=5%3A220109)

## Acceptance Criteria
- When a user has background media added then removing it from the "Page background" panel or via Keyboard delete should result in the Page being selected.

### PR DESCRIPTION
**Pull request by miina**
## Summary
Currently, when moving background media, the media stays selected. This PR changes the selection to Page for keeping the same panels.

## To-do
N/A

## Testing Instructions

1. Add background media
2. Detach media from the background using `X` in the Page background panel.
3. Verify that the Page is selected and the Page background panel is still visible.
### QA
Use the same steps as above.

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6907

### CODE REVIEW

File: assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js [hunk 1]
```diff
@@ -106,7 +106,15 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
       true
     );
     clearBackgroundElement();
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js]: My intention was to make sure that this function, `clearBackgroundElement`, did just that [in the underlying implementation](https://github.com/google/web-stories-wp/blob/54304800b6c6f32836696d1bee8b40d5f0d23024/assets/src/edit-story/app/story/useStoryReducer/reducers/setBackgroundElement.js#L49-L50) - and adding this restriction [to the unit test](https://github.com/google/web-stories-wp/blob/54304800b6c6f32836696d1bee8b40d5f0d23024/assets/src/edit-story/app/story/useStoryReducer/test/clearBackgroundElement.js#L23).

**Review by barklund — CHANGES_REQUESTED** [review]
I think there's a much cleaner way to do this.

File: assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js (hunk 1 above)
Comment (miina) [reply assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js]: I considered this but it doesn't somehow seem like a clean way to do this, it doesn't seem like the action of detaching background element should be aware of what's being selected, no? It seems to be more of a UX choice for the panel that media element should be unselected and Page selected instead after detaching, not something that the reducer should depend on. Thoughts?

Comment (barklund) [reply assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js]: The reducer already manages selection as well and is opinionated on this for many other operations, so I don't see an issue with it being the source of truth for this either.

Comment (miina) [reply assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js]: Ok, fair enough.

File: assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js [hunk 2]
```diff
@@ -92,10 +92,6 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
     [updateCurrentPageProperties]
   );
 
-  const { clearBackgroundElement } = useStory((state) => ({
-    clearBackgroundElement: state.actions.clearBackgroundElement,
-  }));
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/pageBackground/pageBackground.js:97]: Oh, there were two of these? 🤦

**Review by barklund — APPROVED** [review]

**Pull request merged**